docs/concepts/core-service/README.mdx

```diff
@@ -66,44 +65,47 @@ If you want to directly call OIDC APIs, remember to set header `Content-Type: ap
 - [OpenID Connect Core](https://openid.net/specs/openid-connect-core-1_0.html)
 - [OpenID Connect Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html)
 - [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html)
+- [OpenID Connect Back-Channel Logout](https://openid.net/specs/openid-connect-backchannel-1_0-final.html)
+- [OAuth 2.0](https://www.rfc-editor.org/rfc/rfc6749.html)
+- [OAuth 2.0 Token Introspection](https://www.rfc-editor.org/rfc/rfc7662.html)
 - [OAuth 2.0 Token Revocation](https://www.rfc-editor.org/rfc/rfc7009.html)
 - [OAuth 2.0 Resource Indicators](https://www.rfc-editor.org/rfc/rfc8707.html)
+- [Proof Key for Code Exchange (PKCE)](https://www.rfc-editor.org/rfc/rfc7636.html)
+
+## Public API
```

Comment:
Should we use `Public APIs` here? The plural form indicates it's actually a collection of Managment API, experience API and Account API.